MenuXDGCheck.py

```diff
@@ -8,10 +8,7 @@
 
 import codecs
 import os
-try:
-    import ConfigParser as cfgparser
-except ImportError:
-    import configparser as cfgparser
+import configparser as cfgparser
```

Comment:
While openSUSE only runs rpmlint via python 3, upstream still supports python 2.x.. I would prefer if we'd still keep that for now.

Reply:
I'd like to drop Python 2 support. Now is as good of a time as any, as there are no users of the current release with Python 2, and a number of distributions are simply not including Python 2 anymore if they can help it (RHEL 8, SLE 15, OMV Lx 4), while other distributions are working to kill off Python 2 in the next couple of years (Fedora, Mageia, etc.).